lib/mongo/error/operation_failure.rb

```diff
@@ -224,6 +224,14 @@ def write_concern_error?
       # @since 2.10.0
       attr_reader :write_concern_error_code_name
 
+      # @return [ String | nil ] The details of the error.
+      #   For WriteConcernErrors this is `document['writeConcernError']['errInfo']`.
+      #   For WriteErrors this is `document['writeErrors'][0]['errInfo']`.
+      #   For all other errors this is nil.
+      #
+      # @since 2.11.0
```

Comment:
We no longer use `@since` tags and this one isn't accurate, since the functionality will be added in 2.18. You can remove this `@since` tag and, if others are in your way going forward, feel free to get rid of them also.

Reply:
done